### ISSUE TITLE
Competition results page doesn't render venue "markdown" syntax

### ISSUE THREAD

**Issue by timhabermaas** [cubing/worldcubeassociation.org#512]
<img width="619" alt="screen shot 2016-04-19 at 12 44 27 am" src="https://cloud.githubusercontent.com/assets/96054/14622093/f1a12fd8-05c7-11e6-82dc-db252904d6ae.png">

**Comment by timhabermaas** [comment]
Reading it again: This doesn't look like valid markdown.

**Comment by timhabermaas** [comment]
@jfly I guess that syntax was supported by the old PHP code, wasn't it? Do we want to support it?

**Comment by jbcm627** [comment]
Yep, that was syntax from the old/php system. The migration just took care of this. But I think there is still an issue where markdown isn't being processed:
https://www.worldcubeassociation.org/competitions/ShakerHeights2016
![capture](https://cloud.githubusercontent.com/assets/964048/14622526/7e6b1544-0598-11e6-849c-9d31b66605e8.PNG)

**Comment by timhabermaas** [comment]
The `contact` field of that competition literally contains that HTML code. I just tried finding other competitions with a similar problem by running `Competition.all.select { |c| c.contact.try(:include?, '<a')}` and the only competition returned is Shaker Heights 2016. I've fixed this in the production data, should look good now.

Since the other issue has been fixed by the migration I'll close this issue.
